packages/blockchain-wallet-v4/src/network/api/bitpay/index.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios'
+
+const url = 'https://bitpay.com/'
```

Comment:
i'd prefer this to be in wallet options `domains` and then passed in. @plondon thoughts?

Reply:
Yes needs to be in wallet-options. Please create PR here https://github.com/blockchain/wallet-options

Reply:
You just need to change wallet-options-v4.json for dev/staging/prod/testnet

Reply:
switched to wallet options domains in latest commit